Tests/Serialization/ResourceSerializerTests.cs

```diff
@@ -9,25 +9,39 @@
 using Xunit;
 using Xunit.Abstractions;
 using Saule.Queries.Including;
+using Saule.Queries.Pagination;
 
 namespace Tests.Serialization
 {
+    [Trait("Serializer", "ResourceSerializer")]
     public class ResourceSerializerTests
     {
-        private readonly ITestOutputHelper _output;
+        internal readonly ITestOutputHelper _output;
```

Comment:
Maybe you mean `protected`? In any case, if we just remove the old serializer, this should not be necessary anymore.